tests/integration-tests/conftest.py

```diff
@@ -404,6 +404,8 @@ def cfn_stacks_factory(request):
     "sa-east-1": ["sa-east-1a", "sa-east-1c"],
     # m6g.xlarge instances not available in eu-west-1c
     "eu-west-1": ["eu-west-1a", "eu-west-1b"],
+    # provide AZs to enforce networking setup used for EFS test
```

Comment:
Can't we simply perform a boto3 call to list the available AZs in a region and randomly select 2 AZs from there when the mapping is not present in `AVAILABILITY_ZONE_OVERRIDES`?  
I'd like to reduce the number of constraints encoded in `AVAILABILITY_ZONE_OVERRIDES` mappings and use it only as a workaround for capacity and resource unavailability.